src/geofiles/writer/geo_vrml_writer.py

```diff
@@ -112,8 +112,9 @@ def _write(
             vertex_mapping = dict()
             for face in obj.faces:
                 for idx in face.indices:
-                    vertex_mapping[idx] = len(local_vertices)
-                    local_vertices.append(data.get_vertex(idx))
+                    if vertex_mapping.get(idx) == None:
```

Comment:
src/geofiles\writer\geo_vrml_writer.py:115:48: E711 comparison to None should be 'if cond is None:'